wildfly-builder-image/image.yaml

```diff
@@ -1,20 +1,26 @@
 schema_version: 1
 
-name: &imgName "wildfly/wildfly-centos7"
+name: &imgName "wildfly/wildfly-s2i-jdk11"
 description: "The WildFly s2i image"
 version: &imgVersion "dev"
-from: "centos/s2i-base-centos7"
+from: "registry.access.redhat.com/ubi8/ubi-minimal"
 labels:
     - name: io.k8s.description
-      value: "Platform for building and running JEE applications on WildFly $WILDFLY_VERSION"
+      value: "Platform for building and running JEE applications on WildFly"
     - name: io.k8s.display-name
-      value: "WildFly $WILDFLY_VERSION"
+      value: "WildFly"
     - name: io.openshift.expose-services
-      value: "8080:http,8778:jolokia"
+      value: "8080:http"
     - name: io.openshift.tags
-      value: "builder,wildfly,wildfly24"
+      value: "builder,wildfly"
     - name: maintainer
```

Comment:
@jfdenise do you want to be the maintainer or should we move to a community mailing list address (eg `wildfly-dev`)?

Reply:
I set wildfly-dev